toolkits/springer/packages/springer-dropdown/js/dropdown.js

```diff
@@ -171,8 +171,14 @@ const Dropdown = class {
 			this._menuEl.classList.add(this._options.HIDE_CLASS);
 		}
 
-		if (this._options.DROPDOWN_CLASS.length > 0) {
-			this._dropdownEl.classList.add(this._options.DROPDOWN_CLASS);
+		if (this._options.DROPDOWN_CLASSES.length > 0) {
+			// for each classname passed in
+			for (let i = 0; i < this._options.DROPDOWN_CLASSES.length; i++) {
```

Comment:
Any reason to not use `forEach`?

Reply:
On second thoughts, you don't need a loop.
If the option was `DROPDOWN_CLASSLIST` then it could take comma separated list of classes and you wouldn't need to loop through them. You could just `classList.add(DROPDOWN_CLASSLIST)` (because you can do `classList.add('class1', 'class2', 'class3')`

Reply:
I tried the comma separated idea. The trouble is how to do you feed them in to classList as arguments, as you can't feed them in as `"class1, class2, class3"`, you need to feed them in as 3 separate strings with comma separating them. I went ahead with that but once I'd written the code to do all that string processing to achieve that it became clear that is simpler and less code to have an array instead that does multiple calls of `classList.add()`.

Reply:
re `forEach()` - that one baffled me. For some reason that code blows up in Jest tests if you use forEach but is fine in the browser. In Jest test output is says forEach does not exist for `this._options.DROPDOWN_CLASSES` even though if you clog the values going in to it it's clearly an array. It was very weird and after an hour I fell back to a for loop.